### PR TITLE
Added sending unsponsored transaction to kit demo

### PR DESCRIPTION
**Pull request by andrewlee348**

### CODE REVIEW

**Review by SamueleA — DISMISSED** [review]
LGTM!

**Comment by tolgahan-arikan** [comment]
after talking with Andrew we decided to add sponsored call for Polygon as well (he is also adding one for Immutable) so that examples match in all available chains in demo. I'll put this in draft for now.

**Comment by andrewlee348** [comment]
I noticed that the next demo didnt have sponsored transaction testing so I added that while I was at it.

File: examples/next/src/app/components/Connected.tsx [hunk 1]
```diff
@@ -129,7 +144,10 @@ export const Connected = () => {
     if (txnData2) {
       setLastTxnDataHash2((txnData2 as any).hash ?? txnData)
     }
-  }, [txnData, txnData2])
+    if (txnData3) {
+      setLastTxnDataHash2((txnData2 as any).hash ?? txnData)
```
Comment (SamueleA) [inline examples/next/src/app/components/Connected.tsx:148]: I think this should be based on the logic of the previous lines?
```
setLastTxnDataHash>>>3<<<((txnData>>>2<<< as any).hash ?? txnData>>>3<<<)
```

File: examples/next/src/app/components/Connected.tsx [hunk 2]
```diff
@@ -129,7 +144,10 @@ export const Connected = () => {
     if (txnData2) {
       setLastTxnDataHash2((txnData2 as any).hash ?? txnData)
```
Comment (SamueleA) [inline examples/next/src/app/components/Connected.tsx:145]: I think the variable after the ?? should be txnData2, right? (I know, I know this is technically not part of this PR XD)

File: examples/react/src/components/Connected.tsx [hunk 3]
```diff
@@ -156,9 +183,13 @@ export const Connected = () => {
       setLastTxnDataHash((txnData as any).hash ?? txnData)
     }
     if (txnData2) {
-      setLastTxnDataHash2((txnData2 as any).hash ?? txnData)
+      setLastTxnDataHash2((txnData2 as any).hash ?? txnData2)
+    }
+    if (txnData3) {
+      setLastTxnDataHash3((txnData3 as any).hash ?? txnData3)
+      console.log('txnData3', txnData3)
```
Comment (SamueleA) [inline examples/react/src/components/Connected.tsx:190]: Let's remove the console.logs that were added for developing only

**Review by SamueleA — CHANGES_REQUESTED** [review]

File: examples/next/src/app/components/Connected.tsx (hunk 1 above)
Comment (andrewlee348) [reply examples/next/src/app/components/Connected.tsx]: ahh, this was an oversight. The react one was right but I forgot to set it for the next component. Its fixed now

**Review by SamueleA — APPROVED** [review]

**Pull request merged**